docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+.. _Course:
+
+Course
+*******
+
+* count (integer): The number of courses in the catalog.
+* next (string): The URL for the next page of results.
+* previous (string): The URL for the previous page of results.
+* results (array): See :ref:`Results`.
+
+.. _Results:
+
+Results
+*******
+
+Parent: ``course``
+
+A list of courses in the catalog.
+
+* key (string): The unique identifier for the course.
+* title (string): The title of the course.
+* short_description (string): The short description of the course and its
+  content.
+* full_description (string): The long description of the course and its
+  content.
+* level_type (ENUM string): ---DESCRIPTION---
+* subjects (array): Academic subjects that this course covers. See
+  :ref:`Subjects`.
+* prerequisites (array): Any courses a learner must complete before enrolling
+  in the current course. ---This is in strikethrough text in the Google doc. Is
+  it to be deleted?---
+* expected_learning_items (array): ---This is in strikethrough text in the
+  Google doc. Is it to be deleted?---
+* image (array): The About page image for this course. See :ref:`image`.
+* video (array): The course About video. See :ref:`Video`.
+* owners (array): Institution that offers the course. See :ref:`Organization`.
+* sponsors (array): Corporate sponsor for the course. See :ref:`Organization`.
+* modified (datetime): The date and time the course was last modified.
+* course_runs (array): Information about specific runs of the course. See
+  :ref:`Course Runs`.
+* marketing_url: ---DESCRIPTION---
+
+.. _Course Runs:
+
+course_runs
+============
+
+Parent: ``results``
+
+A list of course runs for each course.
+
+* course
+* key (string): The unique identifier for the course.
+* title (string): The title of the course.
+* short_description (string): The short description of the course and its
+  content.
+* full_description (string): The long description of the course and its
+  content.
+* start (datetime): The course start date.
+* end (datetime): The course end date.
+* enrollment_start (datetime): The course enrollment start date.
+* enrollment_end (datetime): The course enrollment end date.
+* announcement ---Not in Google doc---
+* image (array): See :ref:`Image`.
+* video (array): The About video for this course run. See :ref:`Video`.
+* seats (array): The available modes for this course. See :ref:`Seats`.
+* content_language (string): The language for this course run.
+* transcript_languages (array[string]): Languages in which video transcripts
+  are available. ---This is in strikethrough text in the Google doc. Is
+  it to be deleted?---
```

Comment:
ISO codes - like usernames - en-us, fr-fr (IETF language tag - combination of ISO codes for language and region)